{"uuid": "0fb984cd-e169-4ef2-92cb-4e17e85ad2ac", "children": ["f8c01afe-cdbf-4513-ba05-811954c2c78c"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: 'Session' object has no attribute 'testfaied'\n", "trace": "  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\PycharmProjects\\selenium_project_14.09.2023\\page_object_pattern\\tests\\conftest.py\", line 15, in setup\n    before_failed = request.session.testfaied\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1694947097962, "stop": 1694947100246}], "start": 1694947097962, "stop": 1694947100282}